app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,11 @@
+class AdminsController < ApplicationController
+  before_action :require_admin
+
+  def show
+
+  end
+
+  def require_admin
+    render '/../../public/404' unless current_admin?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.